azavea/nasa-hyperspectral
# Planet Query Script

## Overview

A script to allow querying and activation (on the Planet side) of Planet imagery.

Connects #193

## Demo

```bash
./planet-query.py --planet-api-key $(cat planet-api-key.txt ) --datestart 2021-01-01T00:00:00.000Z --dateend 2021-02-01T00:00:00.000Z --no-activate
```

## Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

Run the script without arguments to see usage instructions.  The script is used locally to communicate with the Planet API.

## Checklist
